es66x/src/main/scala/tech/beshu/ror/es/services/HighLevelClientAuditSinkService.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.es.services
+
+import monix.execution.Scheduler
+import org.apache.http.HttpHost
+import org.apache.http.conn.ssl.NoopHostnameVerifier
+import org.apache.http.impl.nio.client.HttpAsyncClientBuilder
+import org.apache.logging.log4j.scala.Logging
+import org.elasticsearch.action.index.{IndexRequest, IndexResponse}
+import org.elasticsearch.client.{RequestOptions, RestClient, RestHighLevelClient}
+import org.elasticsearch.common.xcontent.XContentType
+import tech.beshu.ror.accesscontrol.domain.AuditCluster
+import tech.beshu.ror.es.AuditSinkService
+import tech.beshu.ror.es.utils.GenericResponseListener
+
+import java.security.cert.X509Certificate
+import javax.net.ssl.{SSLContext, TrustManager, X509TrustManager}
+
+class HighLevelClientAuditSinkService(client: RestHighLevelClient)
+                                     (implicit scheduler: Scheduler)
+  extends AuditSinkService
+    with Logging {
+
+  override def submit(indexName: String, documentId: String, jsonRecord: String): Unit = {
```

Comment:
oh, can we add the implicit `RequestContext.Id` parameter to the method? We could use it in logs

Reply:
It seems request context id is passed as `documentId` and it's already logged.